pages/community/[slug].tsx

```diff
@@ -58,17 +97,40 @@ export default function Post({ post, posts, preview }) {
       readProgress.set(v.value.scrollY);
     },
   });
-  if (!router.isFallback && !post?.slug) {
-    return <ErrorPage statusCode={404} />;
-  }
-  if (!post || !post.content) {
-    return ''; // or handle this case differently based on your requirements
-  }
+  useEffect(() => {
+    if (post && post?.content) {
+      const tempDiv = document.createElement("div");
+      tempDiv.innerHTML = post?.content;
+      const avatarImgElement = tempDiv.querySelector(
```

Comment:
why is this required can it be done by some other way because if there will be any change in the class it will break, we can't rely on query selector value

Reply:
Okay for that purpose I have used regex now!